### DISCUSSION TITLE
Dark Mode not specified by TextFields are changing

### DISCUSSION THREAD

**Discussion by theobouwman** [wix/react-native-ui-lib#2751] [Q&A]
This is our setup code for the theme:

_ Colors.loadDesignTokens({
    primaryColor: Colors.blue40
});

const _colors = {
    screenBG: 'white',
    textColor: Colors.grey10,
    moonOrSun: Colors.yellow30,
    mountainForeground: Colors.green30,
    mountainBackground: Colors.green50,
    $backgroundSuccess: Colors.green40,
    $backgroundSuccessLight: Colors.green70 
}

Colors.loadSchemes({
    light: _colors,
    dark: _colors
});

_

So everything should be the same for light and dark mode right?

But here is a screenshot of the light mode:

![image](https://github.com/wix/react-native-ui-lib/assets/16098190/0a0f1c40-a6ef-40ab-9870-08010baa26a8)

And here a screenshot when dark mode is enabled in system settings:

![image](https://github.com/wix/react-native-ui-lib/assets/16098190/73c2972e-1c2d-4db4-b0d6-db7b4e37014b)

So why is this changing?

**Comment by Inbal-Tish** [comment]
@theobouwman I suspect that you did override all relevant design tokens - $textDefault - so it is still effected be the change to dark mode

**Reply by theobouwman** [reply]
Will look into this

**Reply by theobouwman** [reply]
@Inbal-Tish when I remove all colors and nothing changes to the TextField

**Comment by theobouwman** [comment]
I removed `require('react-native-ui-lib/config').setConfig({appScheme: 'default'});` because we dont want to support dark mode right now. But when switching between dark and light mode the TextField placeholder color changes...
